keyboards/rot13labs/veilid_sao/keymaps/default/keymap.c

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2012,2013 gezhaoyou <gezhaoyou@126.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include QMK_KEYBOARD_H
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+    case QK_USER_1:
+        if (record->event.pressed) {            
+            SEND_STRING(SS_DOWN(X_LGUI) SS_TAP(X_SPACE) SS_UP(X_LGUI));
+            _delay_ms(1000);
+            SEND_STRING(SS_TAP(X_LGUI));
+            _delay_ms(1000);
+            SEND_STRING("terminal");
+            _delay_ms(1000);
+            SEND_STRING(SS_TAP(X_ENT));
+            _delay_ms(1000);
+            SEND_STRING("open https://veilid.com" SS_TAP(X_ENT));
+            _delay_ms(1000);
+            SEND_STRING("start https://veilid.com" SS_TAP(X_ENT));
```

Comment:
```suggestion
            SEND_STRING("open https://veilid.com\n");
            _delay_ms(1000);
            SEND_STRING("start https://veilid.com\n");
```

Reply:
@fauxpark OOoooooo, thats awesome! I didn't realize that worked! Thank you!!